components/compliance-service/test_data/audit_reports/2018-03-04_redhat(2)-alpha-nginx(f)-apache(f)-failed.json

```diff
@@ -7,7 +7,7 @@
   "environment": "DevSec Prod Alpha",
   "roles": ["base_linux", "apache_linux", "nginx-hardening-prod", "dot.role"],
   "recipes": ["apache_extras", "apache_extras::harden", "java::default", "nagios::fix"],
-  "end_time": "2018-03-04T09:18:42Z",
+  "end_time": "2018-03-04T09:19:42Z",
```

Comment:
This is the main change. This is making all reports on a different date. The https://github.com/chef/automate/blob/cc266582d838eedb133cdfbbe1bfa36ce28cf820/components/compliance-service/test_data/audit_reports/2018-03-04_redhat(2)-alpha-nginx(f)-apache(s)-failed.json#L10 report before had the same date.